basyx.aasservice/basyx.aasservice-core/src/test/java/org/eclipse/digitaltwin/basyx/aasservice/AasServiceSuite.java

```diff
@@ -69,9 +75,23 @@
 public abstract class AasServiceSuite {
 
 	private static final PaginationInfo NO_LIMIT_PAGINATION_INFO = new PaginationInfo(0, null);
-
+	
 	protected abstract AasService getAasService(AssetAdministrationShell shell);
 
+	private FileRepository fileRepository;
```

Comment:
doesn't need to be a class var

Reply:
I needed to call the fileRepository.find method in the getThumbnail test.

Reply:
Remove field, and use instead getFileRepository() inside class; same pattern as getAasService

Reply:
I kept it like this because I still need to set a default FileRepository in case it's null from the child classes (There is no FileRepository for client implementation).